tox.ini

```diff
@@ -1,6 +1,7 @@
 [tox]
-envlist = py27,py34,py35,py36,py37
+envlist = py27,py35,py36,py37
```

Comment:
Please drop this change.  Despite its having reached upstream end-of-life, Python 3.4 is still extensively used.  I don't think it will be appropriate for us to drop support for 3.4 for at least another couple years.

Reply:
Dropped.